package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "dialog-polyfill",
+  "version": "0.3.0",
```

Comment:
So with Bower, I found that this was actually ignored and it just used the git tags. Is that the case with NPM, or is this actually needed?